Execute CBS' make targets in docker (#5596)
kyma-project/kyma: README.md

  ```
  ├── .github                     # Pull request and issue templates             
  ├── components                  # Source code of all Kyma components
  ├── docs                        # Documentation source files
  ├── installation                # Installation scripts     
  ├── resources                   # Helm charts and Kubernetes resources for the Kyma installation
  ├── tests                       # Acceptance tests
  ├── common                      # Common Go libraries
  ├── scripts                     # Scripts used for building

>   └── tools                       # Source code of utilities used, for example, for the installation and testing

Reviewer: ```suggestion
  └── tools                       # Source code of utilities used, for installation, testing, and more
```
